security/spoofing/README.md

```diff
@@ -0,0 +1,110 @@
+# Workshop Spoofing
+
+✔️ Learn how to create an SMTP server
+
+✔️ Discover how to spoof an email
+
+✔️ Create an evil proxy
+
+## Setup
```

Comment:
Add an intro about spoofing, what is it, what will be covered in this workshop...